dask/dataframe/groupby.py

```diff
@@ -352,7 +387,8 @@ def _groupby_aggregate(
     observed = {"observed": observed} if observed is not None else {}
 
     grouped = df.groupby(level=levels, sort=sort, **observed, **dropna)
-    return aggfunc(grouped, **kwargs)
+    with check_numeric_only_deprecation():
```

Comment:
My guess is this was added to make sure users weren't spammed with a bunch of warnings being emitted from pandas. If that's the case, mind adding a small comment here that mentions that?